draft-ietf-httpbis-priority.md

```diff
@@ -108,101 +114,100 @@ The term control stream is used to describe the HTTP/2 stream with identifier
-multiplexing is the ability to prioritize the sending of information. This was
-an important realization in the design of HTTP/2. Prioritization is a
-difficult problem, so it will always be suboptimal, particularly if one endpoint
-operates in ignorance of the needs of its peer.
-
-HTTP/2 introduced a complex prioritization scheme that uses a combination of
-stream dependencies and weights to describe an unbalanced tree. This scheme has
-suffered from poor deployment and interoperability.
-
-Clients build an HTTP/2 prioritization tree through a series of individual
-stream relationships, which are transferred to the server using HTTP/2 priority
-signals in either of three forms. First, a HEADERS frame with the PRIORITY flag
-set is an explicit signal that includes an Exclusive flag, Stream Dependency
-field, and Weight field. Second, a HEADERS frame with no PRIORITY flag is an
-implicit signal to use the default priority. Third, the PRIORITY frame, which is
-always explicit since it always contains an Exclusive flag, Stream Dependency
-field, and Weight field.
-
-The rich flexibility of tree building is rarely exercised. Experience has shown
-that clients tend to choose a single model optimized for a web use case and
-experiment within the model constraints, or do nothing at all. Furthermore, many
-clients build their prioritization tree in a unique way, which makes it
-difficult for servers to understand their intent and act or intervene
-accordingly.
-
-Many HTTP/2 server implementations do not include support for the priority
-scheme. Some instead favor custom server-driven schemes based on heuristics or
-other hints, such as resource content type or request generation
-order. For example, a server, with knowledge of the document structure, might
-want to prioritize the delivery of images that are critical to user experience
-above other images, but below the CSS files. Since client trees vary, it is
-impossible for the server to determine how such images should be prioritized
-against other responses.
-
-The HTTP/2 scheme allows intermediaries to coalesce multiple client trees into a
+multiplexing is the ability to prioritize the sending of information.
+Prioritization is a difficult problem, so it will always be suboptimal,
+particularly if one endpoint operates in ignorance of the needs of its peer.
+Priority signalling allows endpoints to communicate their own view of priority
+needs, which can be combined with other factors that are considered during the
+peer's prioritization decision making.
+
+RFC 7540 stream priority ({{Section 5.3 of ?RFC7540}}) is a complex system where
+clients signal stream dependencies and weights to describe an unbalanced tree.
+It suffered from poor deployment and interoperability and was deprecated in a
+revision of HTTP/2 ({{HTTP2}}). However, in order to maintain wire
+compatibility, HTTP/2 priority signals are still mandatory to handle (see
```

Comment:
should this ref be RFC 7540 priority signals, instead of H2-bis prio signals? IIUC the signals are no longer defined in H2-bis, which is taken as the "canonical H2" for this doc, so this should be RFC 7540 for consistency IIUC?

Reply:
I disagree, they are still defined via the section that is cited. RFC 7540bis can't remove HEADERS priority bits or the PRIORITY frame. *How* an RFC 7540bis implementation chooses to treat them is going to be up to it.